www-misc/zoneminder/zoneminder-1.34.8.ebuild

```diff
@@ -0,0 +1,178 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# TO DO:
+# * dependencies of unknown status:
+#	dev-perl/Device-SerialPort
+#	dev-perl/MIME-Lite
+#	dev-perl/MIME-tools
+#	dev-perl/PHP-Serialization
+#	virtual/perl-Archive-Tar
+#	virtual/perl-libnet
+#	virtual/perl-Module-Load
+
+EAPI=6
+
+inherit eapi7-ver perl-functions readme.gentoo-r1 cmake-utils depend.apache flag-o-matic systemd
+
+MY_PN="ZoneMinder"
+
+MY_CRUD_VERSION="3.1.0-zm"
+MY_CAKEPHP_VERSION="1.0-zm"
+
+DESCRIPTION="Capture, analyse, record and monitor any cameras attached to your system"
+HOMEPAGE="https://zoneminder.com/"
+SRC_URI="
+	https://github.com/${MY_PN}/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/${MY_PN}/crud/archive/${MY_CRUD_VERSION}.tar.gz -> crud-${MY_CRUD_VERSION}.tar.gz
+	https://github.com/${MY_PN}/CakePHP-Enum-Behavior/archive/${MY_CAKEPHP_VERSION}.tar.gz -> CakePHP-Enum-Behavior-${MY_CAKEPHP_VERSION}.tar.gz
+"
+
+EGIT_SUBMODULES=( * )
```

Comment:
This doesn't seem to be doing anything here.

Reply:
That's an oops. Will remove.